### PR TITLE
Rewrite to ES6

### PR DESCRIPTION
**Pull request by reenko**

### CODE REVIEW

File: src/sidebar/main.js [hunk 1]
```diff
@@ -1,317 +1,298 @@
...
+			 * @private
+			 */
+			this._catberry = window.catberry;
+
+			/**
+			 * Locator in Catberry
+			 * @type {Object|null}
+			 * @private
+			 */
+
```
Comment (kalyanov) [inline src/sidebar/main.js]: Line break is redundant here.

File: src/sidebar/main.js [hunk 2]
```diff
@@ -1,317 +1,298 @@
...
-		if (data.store) {
-			data.store.data = this._clearProto(this._collectedStoreData) || null;
-			data.store.state = currentStateMap[data.store.name] || null;
-		}
+				store: this._element.getAttribute('cat-store') ? {
+					name: this._element.getAttribute('cat-store'),
+					state: null,
+					data: null
+				} : null
+			};
```
Comment (kalyanov) [inline src/sidebar/main.js]: I think, there is a need to add one tab in lines 129-132

File: src/sidebar/panel.js [hunk 3]
```diff
@@ -1,237 +1,224 @@
...
-	});
-};
+	/**
+	 * Renders stores.
+	 */
+	renderStores() {
+		this._renderTableAndCounter(SECTIONS.stores, store => {
+			const componentsList = store.components
+							.map(component => `<div class="component-item"><span>${component.name}</span><button data-id="${component.id}" data-tag="${component.name}">Inspect</button></div>`)
+							.join('');
```
Comment (kalyanov) [inline src/sidebar/panel.js]: There is redundant tabs in lines 91-92

File: src/sidebar/main.js (hunk 2 above)
Comment (pragmader) [inline src/sidebar/main.js]: Nope, it's legit.

**Pull request merged**